lib/codecs/Cargo.toml

```diff
@@ -10,11 +10,13 @@ name = "generate-avro-fixtures"
 path = "tests/bin/generate-avro-fixtures.rs"
 
 [dependencies]
+akin = { version = "0.4", optional = true }
```

Comment:
Generally if it is needed and the license is ok then we are fine to pull it in like you have done here.

However, I don't really see that this crate is adding enough to warrant it's inclusion. It's only used in one place from what I can see. There it would be more idiomatic to just use a simple macro:


```rust
macro_rules! deserialize_impl {
    ($enum:ty) => {
	impl $enum {
	    fn deserialize<'de, D>(deserializer: D) -> Result<Self, D::Error>
	
...

}

deserialize_impl!(Facility);
deserialize_impl!(Severity);
```